ansible/setup.sh

```
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -o nounset -o errexit -o pipefail
+
+# Xuất tất cả biến từ file .env
+ENV_FILE_MAIN=$(pwd)/.env
+if [ -f "$ENV_FILE_MAIN" ]; then
+    set -o allexport
+    source "$ENV_FILE_MAIN"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)